types/concurrently/index.d.ts

```diff
@@ -51,6 +54,11 @@ declare namespace concurrently {
          * the prefix type to use when logging processes output.
          */
         prefix?: 'index' | 'pid' | 'time' | 'command' | 'name' | 'none' | string | undefined;
+        /**
+         * list of chalk colors to use on prefixes.
+         * If there are more commands than colors, the last color will be repeated.
+         */
+        prefixColors?: Array<typeof ChalkColor> | undefined;
```

Comment:
This is ostensibly the same as `CommandObj.prefixColor` on L23; I don't think we should have one typed as a `string` and the other as `typeof ChalkColor`.

Reply:
let me review, thx!

Reply:
amended, thx!